modules/ROOT/pages/as2/AS2-connector-xml-maven-6-0.adoc

```diff
@@ -0,0 +1,72 @@
+= AS2 - XML and Maven Support - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+If you manually code a Mule app in XML either from the Anypoint Studio XML editor or from a text editor,
+you can get access to the <connector> in your app by adding reference statements to your XML Mule flow and in the Apache Maven `pom.xml` file.
+
+[NOTE]
+====
+You can avoid the manual process described here by using Studio. When
+you install the AS2 Connector in Anypoint Studio, Studio automatically populates
+the XML code with the AS2 Connector namespace and schema location, and adds a
+dependency to the `pom.xml` file.
+====
+
+== Add a Namespace for the Connector
+
+Paste the following code inside the `mule` tag of the header
+of your configuration XML:
+[source,xml,linenums]
+----
+http://www.mulesoft.org/schema/mule/as2-mule4
+http://www.mulesoft.org/schema/mule/as2/current/mule-as2-mule4.xsd
+----
+
+This example shows how the namespace statements are placed in the XML:
+[source,xml,linenums]
+----
+[source,xml,linenums]
+<mule xmlns:http="http://www.mulesoft.org/schema/mule/http"
+xmlns:as2="http://www.mulesoft.org/schema/mule/as2-mule4"
+xmlns="http://www.mulesoft.org/schema/mule/core"
+xmlns:doc="http://www.mulesoft.org/schema/mule/documentation"
+	xmlns:spring="http://www.springframework.org/schema/beans"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	    xsi:schemaLocation="http://www.springframework.org/schema/beans
+	    http://www.springframework.org/schema/beans/spring-beans-current.xsd
+        http://www.mulesoft.org/schema/mule/core
+	http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+        http://www.mulesoft.org/schema/mule/as2-mule4
+	http://www.mulesoft.org/schema/mule/as2/current/mule-as2-mule4.xsd">
+      <!-- put your global configuration elements and flows here -->
+</mule>
+
+----
+
+== Add a POM File Dependency
+The `pom.xml` Maven file generated by Anypoint Studio contains dependencies for
+a Mule app. When you code a Mule app manually, include this XML snippet in
+your `pom.xml` file to enable access to the connector. The Mule runtime engine
```

Comment:
```suggestion
your `pom.xml` file to enable access to the connector. Mule runtime engine
```

Reply:
Again, I feel like I have edited this text before; have you implemented the latest boilerplate?